### PR TITLE
add initial models

### PR DESCRIPTION
**Pull request by rmarincola**
Add the initial models for recording incidents. We have a model for incident reasons, and a model to link and incident with its reasons.

## Todo

- [x] add tests

### CODE REVIEW

File: airqo_monitor/models.py [hunk 1]
```diff
@@ -1,5 +1,67 @@
 from django.db import models
 
-# Create your models here.
+
 class Greeting(models.Model):
```
Comment (bweems23) [inline airqo_monitor/models.py]: let's trash this (and the corresponding code in views.py)

File: airqo_monitor/models.py [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+
+
+class Incident(models.Model):
+
+    class Meta:
+        db_table = 'incident'
+
+    created_at = models.DateTimeField("date created", auto_now_add=True)
+    channel_id = models.IntegerField(null=False, db_index=True)
+
```
Comment (bweems23) [inline airqo_monitor/models.py]: maybe a fixed_at or something?

Comment (rmarincola) [reply airqo_monitor/models.py]: yes we should add but i want to add it in a later PR once we've decided on what that flow will look like

File: airqo_monitor/models.py (hunk 1 above)
Comment (rmarincola) [reply airqo_monitor/models.py]: will remove in another pr

**Pull request merged**